Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -244,9 +244,11 @@ def audit_style
     def audit_file
       # Under normal circumstances (umask 0022), we expect a file mode of 644. If
       # the user's umask is more restrictive, respect that by masking out the
-      # corresponding bits. (The also included 0100000 flag means regular file.)
-      wanted_mode = 0100644 & ~File.umask
+      # corresponding bits. (The also-included 0100000 flag means regular file.)
+      # We do not care about the group write bit here, since some brew setups
+      # (e.g., brewdo) make it desirable for it to be set.
```

Comment:
```suggestion
```

Reply:
I removed the bracketed bit here, but if you'd like me to remove everything after the comma in this sentence, just let me know.